### ISSUE TITLE
Consider Swift for the backend

### ISSUE THREAD

**Issue by MP0w** [devlucky/maporable-api#5]
e.g. Vapor https://github.com/vapor/vapor

**Comment by MP0w** [comment]
@larribas what do you think? Swift in the backend is coming 🎉

**Comment by MP0w** [comment]
also we could use google's protocol buffers instead of JSON, to try them out!
There is an implementation by apple already for Swift. ( @zzarcon what do you think? )

**Comment by zzarcon** [comment]
+10000 for protocol buffers 😍 but not sure about benefits of using Swift over Go to be honest. 

What be can actually do it's create a different project and implement the same api with Swift since we have the api defined. 

What do you think?

**Comment by MP0w** [comment]
yep, sounds good. I would do it just to try it out... not real benefits ATM
